### PR TITLE
[1LP][RFR] Test network router details

### PR DESCRIPTION
**Pull request by aljazkosir**
This test creates sandbox with Nuage cloud tenant and its entities. We then navigate to Cloud tenant's network routers through provider, where we compare UI stats with sandbox stats.

This PR depends on: https://github.com/ManageIQ/integration_tests/pull/8150

Please review 3rd commit as the first and the second one are related to other PRs.

\cc @miha-plesko

### CODE REVIEW

**Comment by miha-plesko** [comment]
@digitronik I think this PR is rebased to the PR that you mentioned, that's why first commit is repeated. Once you guys merge the dependent PR, the first commit will automatically disappear from here.

That being said, could you just give us review on the second commit here so we can merge faster then, please🙂

File: cfme/tests/networks/nuage/test_inventory.py [hunk 1]
```diff
@@ -67,3 +67,18 @@ def test_l2_subnet_details(setup_provider_modscope, provider, with_nuage_sandbox
...
+
+
+def test_network_router_details(setup_provider_modscope, provider, with_nuage_sandbox_modscope):
+    sandbox = with_nuage_sandbox_modscope
+    tenant_name = sandbox['enterprise'].name
+    router_name = sandbox['domain'].name
+    tenant = provider.collections.cloud_tenants.instantiate(name=tenant_name)
+    router = tenant.collections.routers.instantiate(name=router_name)
+
+    router.validate_stats({
```
Comment (digitronik) [inline cfme/tests/networks/nuage/test_inventory.py]: :+1:

**Comment by mshriver** [comment]
Since this includes commits that are in other open PRs, it should not be marked RFR. The dependent PRs need to be merged first.

**Comment by aljazkosir** [comment]
@mshriver Sorry for the delay. This PR is now rebased with master.

**Pull request merged**